src/static/redirects.njk

```diff
@@ -18,24 +18,25 @@ eleventyExcludeFromCollections: true
 # Internal Redirects
 /demo/*                             /play/:splat 302!
 
-{% if site.language.code == "en" %}
+{% if site.locals.docs %}
 
 # Old-Style Docs to New-Style Docs
 /docs/rules/*                       /docs/latest/rules/:splat 301!
 /docs/user-guide/*                  /docs/latest/user-guide/:splat 301!
 /docs/maintainer-guide/*            /docs/latest/maintainer-guide/:splat 301!
 /docs/developer-guide/*             /docs/latest/developer-guide/:splat 301!
 
-# Version-Specific Docs
-/docs/head/*                        https://docs-eslint.netlify.app/:splat 200!
-/docs/latest/*                      https://latest--docs-eslint.netlify.app/:splat 200!
+# Regular Docs
+/docs/head/*                        https://{{ site.locals.docs }}/:splat 200!
+/docs/latest/*                      https://latest--{{ site.locals.docs }}/:splat 200!
```

Comment:
We should create `latest` branch in https://github.com/eslint/zh-hans.docs.eslint.org and configure Netlify to deploy it?

Reply:
Yup, that was on my todo list.

Although, I think maybe these docs will only ever be translated from `latest`, so maybe we should always have `/docs/head` go to the English site?

Reply:
I think the Chinese translation should be `/docs/head` and `/docs/latest` should go to English site. Because Chinese can never be up to date, I sync each version as it is released.

Reply:
> Although, I think maybe these docs will only ever be translated from `latest`, so maybe we should always have `/docs/head` go to the English site?

Makes sense to me :+1:

Reply:
> I think the Chinese translation should be `/docs/head` and `/docs/latest` should go to English site. Because Chinese can never be up to date, I sync each version as it is released.

`/docs/head` relates to the latest development (i.e., the eslint/eslint `main` branch), and those docs may contain some features that haven't been released yet, so most users should rather be interested in the `/docs/latest` version. It would be confusing if `/docs/head` on different sites has a different meaning, like "the latest translated version".

`/docs/latest` should ideally be the latest published version with the `latest` tag. There will obviously be a little lag on translated sites, but the version selector box makes it clear which particular version the docs are describing.

Reply:
I agree. We should stick with HEAD meaning the latest unreleased stuff while “latest” is the most recent published release (which matches the npm definition as well).